app/controllers/concerns/course/assessment/question/programming/python/package_concern.rb

```diff
@@ -0,0 +1,128 @@
+# frozen_string_literal: true
```

Comment:
Yes, call it package generation service or something, so that you can properly test the service.